SDPi_Supplement/asciidoc/volume3/tf3-ch-8.2.8-biceps-content.adoc

```diff
@@ -4,11 +4,31 @@ NOTE:  This file contains the content that profiles BICEPS semantics for use in
 
 // 8.2.8
 [sdpi_offset=8]
-===== SDC/BICEPS semantic content
+==== SDC/BICEPS semantic content
 
 // 8.2.8.1
 ===== SDC/BICEPS Content Module
 
+====== BICEPS Extension Model
+
+IEEE 11073-10207 provides a powerful extension model.
+This enables manufacturers to add information that does not fit into the defined structure.
+Therefore, nearly every element in the participant and message model contains an extension point (`tns:Extension`).
+However, the fundamental rule is: Do not use an extension for anything that can be expressed using the given models.
+Especially, the powerful concept of coded values must be considered before using an extension.
```

Comment:
We should elaborate on this one - how does it relate to extension containers? And perhaps also avoid to use the word "must" as it implicates a requirement.